controllers/users.controller.js

```diff
@@ -0,0 +1,11 @@
+const { selectUsers } = require("../models/users.model");
+
+exports.getUsers = (req, res, next) => {
+  selectUsers()
+    .then((users) => {
+      res.status(200).send({ users });
+    })
+    .catch((err) => {
```

Comment:
we only need a catch block if we are throwing errors (Promise.reject() in the model for example).  If we aren't we don't need it